<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-17T23:21:37Z" name="Default suite" finished-at="2021-12-17T23:21:46Z" duration-ms="9030">
    <groups>
    </groups>
    <test started-at="2021-12-17T23:21:37Z" name="Default test" finished-at="2021-12-17T23:21:46Z" duration-ms="9030">
      <class name="testng.PhpTravels1">
        <test-method is-config="true" signature="preconditions()[pri:0, instance:testng.PhpTravels1@37e4d7bb]" started-at="2021-12-17T18:21:37Z" name="preconditions" finished-at="2021-12-17T18:21:40Z" duration-ms="3029" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preconditions -->
        <test-method signature="openurl()[pri:0, instance:testng.PhpTravels1@37e4d7bb]" started-at="2021-12-17T18:21:40Z" name="openurl" finished-at="2021-12-17T18:21:44Z" duration-ms="3548" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openurl -->
        <test-method signature="clicklogin()[pri:1, instance:testng.PhpTravels1@37e4d7bb]" started-at="2021-12-17T18:21:44Z" name="clicklogin" finished-at="2021-12-17T18:21:45Z" depends-on-methods="testng.PhpTravels1.openurl" duration-ms="572" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clicklogin -->
        <test-method signature="enterusernameandpassword()[pri:2, instance:testng.PhpTravels1@37e4d7bb]" started-at="2021-12-17T18:21:45Z" name="enterusernameandpassword" finished-at="2021-12-17T18:21:46Z" depends-on-methods="testng.PhpTravels1.clicklogin" duration-ms="1844" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dashboard - PHPTRAVELS] but found [Login - PHPTRAVELS]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dashboard - PHPTRAVELS] but found [Login - PHPTRAVELS]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at testng.PhpTravels1.enterusernameandpassword(PhpTravels1.java:67)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterusernameandpassword -->
        <test-method signature="clicklogout()[pri:3, instance:testng.PhpTravels1@37e4d7bb]" started-at="2021-12-17T18:21:46Z" name="clicklogout" finished-at="2021-12-17T18:21:46Z" depends-on-methods="testng.PhpTravels1.enterusernameandpassword" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method PhpTravels1.clicklogout()[pri:3, instance:testng.PhpTravels1@37e4d7bb] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method PhpTravels1.clicklogout()[pri:3, instance:testng.PhpTravels1@37e4d7bb] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clicklogout -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:testng.PhpTravels1@37e4d7bb]" started-at="2021-12-17T18:21:46Z" name="closebrowser" finished-at="2021-12-17T18:21:46Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closebrowser -->
      </class> <!-- testng.PhpTravels1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
